src/main/java/edu/cornell/kfs/module/purap/businessobject/options/OrderCompletedValuesFinder.java

```diff
@@ -7,14 +7,16 @@
 import org.kuali.kfs.core.api.util.KeyValue;
 import org.kuali.kfs.krad.keyvalues.KeyValuesBase;
 
+import edu.cornell.kfs.fp.CuFPConstants;
+
 public class OrderCompletedValuesFinder extends KeyValuesBase {
 	
 	private static final long serialVersionUID = 1L;
 
 	public List<KeyValue> getKeyValues() {
         List<KeyValue> keyValues = new ArrayList<KeyValue>();
-        keyValues.add(new ConcreteKeyValue("N", "College/Unit approved for <b>service center processing.</b> (Select and click Approve to send to service center for processing.)"));
-        keyValues.add(new ConcreteKeyValue("Y", "College/Unit has processed this request. (Select and click Approve to finalize this request.)"));
+        keyValues.add(new ConcreteKeyValue(CuFPConstants.NO, "College/Unit approved for <b>service center processing.</b> (Select and click Approve to send to service center for processing.)"));
+        keyValues.add(new ConcreteKeyValue(CuFPConstants.YES, "College/Unit has processed this request. (Select and click Approve to finalize this request.)"));
```

Comment:
Although these constants are from an FP class, I agree that their names are much more concise and generic than any existing non-module-specific equivalents in our code or in base code. I would suggest creating a follow-up user story to get these yes/no constants added to `CUKFSConstants`, so that they'll be in a better location for multi-module use.

Reply:
created https://jira.cornell.edu/browse/KFSPTS-28602